tests/test_workspace_db_special_paths.py

```
@@ -57,13 +57,13 @@ def test_build_composer_id_to_workspace_id_handles_spaces(self):
         with tempfile.TemporaryDirectory() as tmp:
             ws_root = self._build_fixture(tmp)
             entries = [{"name": "ws-with spaces", "workspaceJsonPath": ""}]
-            mapping = _build_composer_id_to_workspace_id(ws_root, entries)
+            mapping = build_composer_id_to_workspace_id(ws_root, entries)
             self.assertEqual(mapping, {"cid-space": "ws-with spaces"})
 
     def test_open_global_db_handles_spaces(self):
         with tempfile.TemporaryDirectory() as tmp:
             ws_root = self._build_fixture(tmp)
-            with _open_global_db(ws_root) as (conn, path):
+            with open_global_db(ws_root) as (conn, path):
```

Ruff (0.15.14)

[warning] Unpacked variable `path` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)